src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -224,15 +223,109 @@ public override void VisitNamedType(NamedTypeSymbol symbol)
                 return;
             }
 
-            DefaultVisit(symbol);
+            // 1. visit the type
+            if (symbol.IsExtension && (SourceNamedTypeSymbol)symbol.ContainingType is { } containingType)
+            {
+                ImmutableArray<SourceNamedTypeSymbol> extensions = containingType.GetExtensionGroupingInfo().GetMatchingExtensions((SourceNamedTypeSymbol)symbol);
+                appendMergedExtensionBlocks(extensions);
```

Comment:
>appendMergedExtensionBlocks [](http://example.com/codeflow?start=16&length=27)

It would be good to add a comment about scenario when this code path is reachable.